cla_public/static-src/javascripts/modules/form-errors.js

```diff
@@ -262,11 +262,19 @@
       _.each(errorFields, addErrors);
 
       if(this.$form.data('error-banner') !== false) {
-        this.$form.closest('main').prepend(this.mainFormError({ errors: this.createErrorSummary(unattachedErrors)}));
-        if (GOVUK.getCookie("locale") == "cy_GB") {
-          $("title").prepend("Gwall: ");
+        if ($("#submit-button").length && $(".s-hidden input").val()) {
+          // In case a hidden field is throwing an error (and there is a submit button),
+          $(".s-hidden input").val("");
+          // we wipe the value,
+          $("#submit-button").click();
+          // and retry the submit.
         } else {
-          $("title").prepend("Error: ");
+          this.$form.closest('main').prepend(this.mainFormError({ errors: this.createErrorSummary(unattachedErrors)}));
+          if (GOVUK.getCookie("locale") == "cy_GB") {
+            $("title").prepend("Gwall: ");
+          } else {
+            $("title").prepend("Error: ");
```

Comment:
This else statement doesn't seem to do anything as it already get sets to Error by default?

Reply:
This is the page title.  It doesn't have Error to begin with and since this is a prepend, if it did it would result in `Error: Error:`

Reply:
Ah it's the page title 🤦 . No worries ty